data/Dockerfiles/dovecot/imapsync_cron.pl

```diff
@@ -19,11 +19,20 @@
 }
 
 sub qqw($) {
-  my @values = split('(?=--)', $_[0]);
+  my @params = ();
+  my @values = split(/(?=--)/, $_[0]);
   foreach my $val (@values) {
+    my @tmpparam = split(/ /, $val, 2);
+    foreach my $tmpval (@tmpparam) {
```

Comment:
This is exactly the Perl magic I was trying to implement for like 2 hours now... I am not good at Perl.

I am not good at Python either. Or PHP. Perl is just worst.

Reply:
Perl is also not my favourite. I had to remove the old dust on my knowledge. ;-)